Azure/sonic-utilities
# [config][generic-update] Implementing patch sorting (2nd attempt)

#### What I did
Implemented [JSON Patch Ordering using YANG Models Design Doc](https://github.com/Azure/SONiC/blob/master/doc/config-generic-update-rollback/Json_Patch_Ordering_using_YANG_Models_Design.md)

#### How to verify it
Unit-Tests

**NOTE: The code in this PR was [reverted](github.com/Azure/sonic-utilities/commit/0a145e8027380e8d4decb36bdfc647062c722612) before because of some [build issues](https://github.com/Azure/sonic-utilities/issues/1761). Build issues have been fixed [here](https://github.com/Azure/sonic-buildimage/pull/8632). To check the original PR comments please go [here](https://github.com/Azure/sonic-utilities/pull/1599).**
